src/forklift/cli.py

```diff
@@ -6,32 +6,32 @@
 A module that contains the implementation of the cli commands
 '''
 
+import core
+import lift
 import logging
+import seat
 import settings
 import sys
 from glob import glob
 from json import dumps, loads
-from os.path import abspath, exists, join, splitext, basename, dirname
 from models import Pallet
-import lift
-import core
+from os.path import abspath, exists, join, splitext, basename, dirname
+from time import clock
 
 log = logging.getLogger(settings.LOGGER)
 
 
 def init():
     if exists('config.json'):
-        return 'config file already created.'
+        return abspath('config.json')
```

Comment:
I think that this should return some message indicating that the config file was already created. Otherwise the output is exactly the same between creating a new one and a no-op for an existing one.